ntbs-service/Pages/ClinicalTimelines/Edit.cshtml.cs

```diff
@@ -0,0 +1,79 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.DataAccess;
+using ntbs_service.Helpers;
+using ntbs_service.Models;
+using ntbs_service.Pages;
+
+namespace ntbs_service.Pages_ClinicalTimelines
+{
+    public class EditModel : ValidationModel
+    {
+        private readonly INotificationRepository _repository;
+
+        public EditModel(INotificationRepository repository)
+        {
+            _repository = repository;
+        }
+
+        [BindProperty]
+        public ClinicalTimeline ClinicalTimeline { get; set; }
+        [BindProperty]
+        public int NotificationId { get; set; }
+
+        [BindProperty]
+        public FormattedDate FormattedSymptomDate { get; set; }
+        [BindProperty]
+        public FormattedDate FormattedPresentationDate { get; set; }
+        [BindProperty]
+        public FormattedDate FormattedDiagnosisDate { get; set; }
+        [BindProperty]
+        public FormattedDate FormattedTreatmentDate { get; set; }
+        [BindProperty]
+        public FormattedDate FormattedDeathDate { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int? id)
+        {
+            var notification = await _repository.GetNotificationAsync(1);

```

Comment:
I was just looking through the code for referencing. is 1 as id was used for testing purpose or is it meant to be 1 until we implement changes to Notification structure?

Reply:
It was just a quick fix before integrating into the proper workflow - will now do this by using the notification id from the patient details screen